CorePush/Utils/JsonHelper.cs

```diff
@@ -6,8 +6,7 @@ namespace CorePush.Utils
     public static class JsonHelper
     {
         private static readonly JsonSerializerSettings settings = new JsonSerializerSettings
-        {
-            ContractResolver = new CamelCasePropertyNamesContractResolver(),
```

Comment:
Unfortunately this introduces a breaking change. We can't do it. May consider it in the next major version